src/Logikfabrik.Umbraco.Jet/Mappings/DataTypeDefinition.cs

```diff
@@ -25,6 +25,11 @@ public enum DataTypeDefinition
         ContentPicker = 1034,
 
         /// <summary>
+        /// Data type for new content picker.
+        /// </summary>
+        ContentPicker2 = 1046,
```

Comment:
Ah, I see! OK, then I think it's best to stick with the same names as in Umbraco and keep the 2.